simplivity/resources/resource.py

```diff
@@ -212,15 +212,19 @@ def task_affected_resources(self, task, timeout):
 
         return affected_resources
 
-    def do_get(self, uri):
+    def do_get(self, uri, filters=None):
         """Makes get requests
 
         Args:
             uri: URI of the resource
+            flags: Dictionary of filters, example: {'name': 'name'}
```

Comment:
Variable name is different in the doc section then the name passed in the method